include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -485,11 +488,134 @@ struct __radix_sort_scan_submitter<_RadixLocalScanName, __internal::__optional_k
     }
 };
 
+struct __empty_peer_temp_storage
+{
+    template <typename... T>
+    __empty_peer_temp_storage(T&&...)
+    {
+    }
+};
+
+enum class __peer_prefix_algo
+{
+    subgroup_ballot,
+    atomic_fetch_or,
+    scan_then_broadcast
+};
+
+template <typename _OffsetT, __peer_prefix_algo _Algo>
+struct __peer_prefix_helper;
+
+template <typename _OffsetT>
+struct __peer_prefix_helper<_OffsetT, __peer_prefix_algo::atomic_fetch_or>
+{
+    using _AtomicT = sycl::atomic_ref<::std::uint32_t, sycl::memory_order_relaxed, sycl::memory_scope::sub_group,
```

Comment:
PR #683 actually already replaces the `sycl::atomic_ref` with `__dpl_sycl::__atomic_ref`. This was required to support older compilers.

Reply:
In https://github.com/oneapi-src/oneDPL/pull/684 I made some modifications of __dpl_sycl::__atomic_ref and it's usage

Reply:
Sergey's patch #684 will need to be applied anyway, as it "fixes" the DPL's atomic_ref to be more like SYCL 2020 one than the current code. It's OK to add it after the release, but then its use in radix-sort will require a small change again. Though, since you have already merged the branch, perhaps Sergey can rebase it and make that small change.